System/src/Check.cs

```diff
@@ -13,173 +13,176 @@ namespace Wangkanai;
 [DebuggerStepThrough]
 public static class Check
 {
-	#region ThrowIfNull
+	#region Throw
 
-	[ContractAnnotation(AnnotationResources.ValueNullThenHalt)]
```

Comment:
Remove to make the code clean. This is [JetBrains Contract Annotation ](https://www.jetbrains.com/help/resharper/Reference__Code_Annotation_Attributes.html#ContractAnnotationAttribute) and its make harder to read.